_includes/tutorials/session-windows/ksql/code/tutorial-steps/dev/populate-stream.sql

```diff
@@ -1,9 +1,10 @@
-INSERT INTO clicks (ip, timestamp, url) VALUES ('51.56.119.117','2019-07-18T10:00:00Z','/etiam/justo/etiam/pretium/iaculis.xml');
-INSERT INTO clicks (ip, timestamp, url) VALUES ('51.56.119.117','2019-07-18T10:01:00Z','/nullam/orci/pede/venenatis.json');
-INSERT INTO clicks (ip, timestamp, url) VALUES ('53.170.33.192','2019-07-18T10:01:31Z','/mauris/morbi/non.jpg');
-INSERT INTO clicks (ip, timestamp, url) VALUES ('51.56.119.117','2019-07-18T10:01:36Z','/convallis/nunc/proin.jsp');
-INSERT INTO clicks (ip, timestamp, url) VALUES ('53.170.33.192','2019-07-18T10:02:00Z','/vestibulum/vestibulum/ante/ipsum/primis/in.json');
-INSERT INTO clicks (ip, timestamp, url) VALUES ('51.56.119.117','2019-07-18T11:03:21Z','/vehicula/consequat/morbi/a/ipsum/integer/a.jpg');
-INSERT INTO clicks (ip, timestamp, url) VALUES ('51.56.119.117','2019-07-18T11:03:50Z','/pede/venenatis.jsp');
-INSERT INTO clicks (ip, timestamp, url) VALUES ('53.170.33.192','2019-07-18T11:40:00Z','/nec/euismod/scelerisque/quam.xml');
-INSERT INTO clicks (ip, timestamp, url) VALUES ('53.170.33.192','2019-07-18T11:40:09Z','/ligula/nec/sem/duis.jsp');
+INSERT INTO clicks (ip, timestamp, url) VALUES ('51.56.119.117',FORMAT_TIMESTAMP(FROM_UNIXTIME(UNIX_TIMESTAMP()),'yyyy-MM-dd''T''HH:mm:ssX'),'/etiam/justo/etiam/pretium/iaculis.xml');
+INSERT INTO clicks (ip, timestamp, url) VALUES ('51.56.119.117',FORMAT_TIMESTAMP(FROM_UNIXTIME(UNIX_TIMESTAMP() + (60 * 1000)),'yyyy-MM-dd''T''HH:mm:ssX'),'/nullam/orci/pede/venenatis.json');
```

Comment:
The tutorial needs timestamps at different intervals to create sessions.  The idea is to add time in milliseconds to the current time - here the format is `Seconds * 1000`